.spelling

```diff
@@ -0,0 +1,724 @@
+100ms
+xUnit
+Api
+Runspace
+MessageAnalyzer
+TabCompletion
+Interop
+DotNet
+NativeExecution
+Microsoft.PowerShell.Archive
+Microsoft.PowerShell.Core
+Microsoft.PowerShell.Diagnostics
+Microsoft.PowerShell.Management
+Microsoft.PowerShell.Security
+Microsoft.PowerShell.Utility
+ - docs/testing-guidelines/TestRoadmap.md
+Syslog
+OpenCover
+corefx
+DotCover
+loopback
+WSMan
+Downlevel
+KPIs
+PowerBI
+MVPs
+org
+ - demos/Apache/readme.md
+Get-ApacheVHost
+Get-ApacheModule
+Restart-ApacheHTTPserver
+New-ApacheVHost
+ - docs/testing-guidelines/WritingPesterTests.md
+FullyQualifiedErrorId
+FQErrorId
+TestDrive
+Get-Childitem
+PSDefaultParameterValues-skip
+nGet-MultiLineString
+nGet-ContentOut-String
+StreamDescribeCIFeatureScenarioDescribeContextItContextContextBeforeAllAfterAllBeforeEachAfterEachshould
+throw-testcasesItMockDescribe
+ShouldShouldIt
+ - README.md
+opencode
+microsoft.com
+ - src/libpsl-native/README.md
+libpsl-native
+C#'s
+codepage
+Ansi
+ - src/Microsoft.PowerShell.PSReadLine/en-US/PSReadline.md
+Microsoft.PowerShell.KeyHandler
+_PSReadline
+coloring
+ScriptBlock
+ConsoleKeyInfo
+BriefDescription
+HistorySearchBackward
+EditMode
+ContinuationPrompt
+ContinuationPromptForegroundColor
+ConsoleColor
+ContinuationPromptBackgroundColor
+EmphasisForegroundColor
+EmphasisBackgroundColor
+ErrorForegroundColor
+ErrorBackgroundColor
+HistoryNoDuplicates
+AddToHistoryHandler
+Func
+ValidationHandler
+ValidateAndAcceptLine
+ToString
+HistorySearchCursorMovesToEnd
+MaximumHistoryCount
+Int32
+MaximumKillRingCount
+ResetTokenColors
+ShowToolTips
+ExtraPromptLineCount
+DingTone
+BellStyle
+DingDuration
+50ms
+CompletionQueryItems
+WordDelimiters
+ForwardWord
+KillWord
+HistorySearchCaseSensitive
+ReverseSearchHistory
+HistorySaveStyle
+SaveIncrementally
+SaveAtExit
+SaveNothing
+HistorySavePath
+AppData
+host.Name
+_history.txt
+TokenKind
+TokenClassification
+ForegroundColor
+BackgroundColor
+ - src/Microsoft.PowerShell.SDK/README.md
+metapackage
+project.json
+ - src/Modules/README.md
+ps1xml
+psm1
+psd1
+ - src/Modules/Shared/Pester/CHANGELOG.md
+GH-362
+GH-368
+TestName
+Invoke-Pester's
+BeforeEach
+AfterEach
+GH-326
+BeOfType
+GH-327
+PSv3
+PSCommandPath
+GH-333
+GH-322
+GH-323
+Assert-MockCalled
+GH-324
+ExclusiveFilter
+ParameterFilter
+OutputXml
+EnableExit
+GH-281
+PassThru
+CodeCoverage
+GH-341
+GH-346
+v2
+ArgumentList
+GH-354
+GH-358
+ExecutionContext
+GH-307
+GetDynamicParameters
+GH-295
+MyInvocation
+GH-304
+Get-MockDynamicParameters
+dynamicparam
+GH-306
+GH-267
+GH-266
+Tests.ps1.
+GH-272
+StrictMode
+GH-274
+InModuleScope
+GH-276
+GH-254
+ModuleName
+GH-278
+GH-290
+ISESteroids
+GH-249
+BeforeAll
+AfterAll
+ErrorAction
+v3
+BeGreaterThan
+BeLessThan
+GH-223
+GH-232
+ExcludeTagFilter
+GH-234
+FunctionName
+GH-215
+OutputFile
+OutputFormat
+LegacyNUnitXml
+GH-200
+GH-203
+GH-209
+nuget.exe
+BeExactly
+GH-192
+GH-195
+GH-149
+GH-150
+Validate-Xml
+GH-152
+GH-155
+GH-156
+GH-158
+GH-163
+GH-164
+GH-165
+GH-166
+GH-167
+GH-147
```

Comment:
Please, add a section about spellchecker to https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#contributing-to-documentation
It could (and probably should) be references to external resources, but there should be clear scenarios:

- How to run locally
- How to bulk-update ignore list